pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -4540,3 +4542,269 @@ func TestDisallowsInvalidFormatOptions(t *testing.T) {
 		}
 	}
 }
+
+func makeBuiltinOverride(
+	builtin *tree.FunctionDefinition, overloads ...tree.Overload,
+) *tree.FunctionDefinition {
+	props := builtin.FunctionProperties
+	return tree.NewFunctionDefinition(
+		"import."+builtin.Name, &props, overloads)
```

Comment:
I don't know what the implications of adding a qualification here are. Maybe it's no problem.